src/pages/docs/TutorialTypescript.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import Typography from "@material-ui/core/Typography";
+import {Provider} from 'react-redux';
+import {person} from '@jsonforms/examples';
+import Demo from "../../common/Demo";
+import {createJsonFormsStore} from "../../common/store";
+import DocPage from "./DocPage";
+/* eslint import/no-webpack-loader-syntax: off */
+const init = require('!raw-loader!./listings/init.md');
+const installation = require('!raw-loader!./listings/installationYarn.md');
+const setupStore = require('!raw-loader!./listings/setupStore.md');
+const render = require('!raw-loader!./listings/renderTypescript.md');
+const initAction = require('!raw-loader!./listings/initAction.md');
+const imports = require('!raw-loader!./listings/imports.md');
+const variables = require('!raw-loader!./listings/variables.md');
+
+const store = createJsonFormsStore({
+  data: person.data,
+  schema: person.personCoreSchema,
+  uischema: person.uischema
+});
+
+const Tutorial = () => (
+  <DocPage>
+    <Typography variant='h1'>
+      Tutorial
+    </Typography>
+    <Typography variant='body1'>
+      This section describes you to integrate JSON Forms into a React app from scratch.
```

Comment:
This is also wrong in the already existing documetnation. It should read: describes [how] you or 'describes [to] you'